### PR TITLE
Cleanup logs and add debug logs in aggregation service

### PR DESCRIPTION
**Pull request by kavyamani596**

### CODE REVIEW

**Review by arahimanshaik — APPROVED** [review]

**Review by mohamed-shahid-afrid — APPROVED** [review]

File: svc-aggregation/main.go [hunk 1]
```diff
@@ -40,6 +41,13 @@ type Schema struct {
 	QueryKeys     []string `json:"queryKeys"`
 }
 
+const (
```
Comment (amar-shalgar) [inline svc-aggregation/main.go]: action id and name should be in constant under lib directories

File: svc-aggregation/main.go [hunk 2]
```diff
@@ -84,7 +92,9 @@ func main() {
 		AddConnectionMethodInterface: agmodel.AddConnectionMethod,
 		DeleteInterface:              agmodel.Delete,
 	}
-	if err := connectionMethodInterface.AddConnectionMethods(config.Data.ConnectionMethodConf); err != nil {
+	aggTransactionID := uuid.New()
+	aggCtx := agcommon.CreateContext(aggTransactionID.String(), aggActionID, aggActionName, "1", common.AggregationService, podName)
```
Comment (amar-shalgar) [inline svc-aggregation/main.go]: what exactly is "1" here?

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: svc-aggregation/main.go (hunk 2 above)
Comment (kavyamani596) [reply svc-aggregation/main.go]: this is the thread ID

File: lib-utilities/common/constants.go [hunk 3]
```diff
@@ -150,6 +150,14 @@ const (
 	AggregationSourceType = "#AggregationSource.v1_2_0.AggregationSource"
 	//ChassisType has version to be returned with Chassis Service
 	ChassisType = "#Chassis.v1_20_0.Chassis"
+	// Aggregation service Action ID
```
Comment (amar-shalgar) [inline lib-utilities/common/constants.go]: Look at the previous implementation for action id and name

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**